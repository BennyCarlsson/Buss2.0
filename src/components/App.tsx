import React from "react"
import TimeBox from "./TimeBox/TimeBox"

function App(): JSX.Element {
  return (
    <div className="App">
      <TimeBox time="12:00" />
    </div>
  )
}

export default App
