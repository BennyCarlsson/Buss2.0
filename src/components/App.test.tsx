import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"
import App from "./App"

it("renders learn react link", () => {
  const { getByText } = render(<App />)
  const linkElement = getByText(/learn react/i)
  expect(linkElement).toBeInTheDocument()
})

it("render App component", () => {
  const tree = renderer.create(<App />).toJSON()
  expect(tree).toMatchSnapshot()
})
