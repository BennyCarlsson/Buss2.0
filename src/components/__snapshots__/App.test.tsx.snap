// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App component 1`] = `
<div
  className="App"
>
  <div
    className="sc-bdnylx boKVLG"
  >
    12:00
  </div>
</div>
`;
